src/render/processor.rs

```diff
@@ -38,6 +38,18 @@ impl std::fmt::Debug for RenderScope {
 }
 
 impl RenderScope {
+    /// Send a message to the corresponding AudioWorkletNode of this processor
+    ///
+    /// This method is just a shim of the full
+    /// [`MessagePort`](https://webaudio.github.io/web-audio-api/#dom-audioworkletprocessor-port)
+    /// `postMessage` functionality of the AudioWorkletProcessor.
+    pub fn post_message(&self, msg: Box<dyn Any + Send + 'static>) {
```

Comment:
I have made the Box explicit here, so the user is forced to think about the allocation that occurs